### PR TITLE
Implement `Base.eps(v)` for `v::Vec{N,<:FloatingTypes}` and `v::Type{Vec{N,<:FloatingTypes}}`

### PR DESCRIPTION
**Pull request by favba**
This PR implements the `Base.eps` function for `Vec`s of Floats.
The algorithm used is the same as from `Base`, with minor adaptation for `Vec`s.

### CODE REVIEW

File: test/runtests.jl [hunk 1]
```diff
@@ -304,6 +304,17 @@ llvm_ir(f, args) = sprint(code_llvm, f, Base.typesof(args...))
         # See: https://github.com/eschnett/SIMD.jl/pull/43
         ir = llvm_ir(^, (V4F64(v4f64), Int32(2)))
         @test occursin("@llvm.powi.v4f64", ir)
+
+        #tests for eps implementation
+        let v = V4F64((1, -1000, 100000, -1000000000))
```
Comment (eschnett) [inline test/runtests.jl]: Edge cases can be tricky. Can you add tests for prevfloat(1.0), nextfloat(1.0), as well as 1.5, prevfloat(1.5), and nextfloat(1.5)?

Comment (favba) [reply test/runtests.jl]: Done. Also added a more explicit sanity check

**Comment by eschnett** [comment]
Is there some documentation that needs to be updated? There may be a list of supported functions somewhere.

**Comment by favba** [comment]
> Is there some documentation that needs to be updated? There may be a list of supported functions somewhere.

Done!
